### PR TITLE
json read and write

### PR DESCRIPTION
**Pull request by santhosh-as-rsk**

### CODE REVIEW

File: json.feature [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Feature: To write and read a json file
+
+Scenario:
```
Comment (SubithaMS) [inline json.feature]: add the description here

File: json.feature [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    * def jsonObject = {name: 'santhosh', age: 18}
+    * def data = read('new.json')
+    * print 'data', data
+    * def value =
```
Comment (SubithaMS) [inline json.feature]: give a meaningful name for the function

File: json_write.feature [hunk 3]
```diff
@@ -0,0 +1,21 @@
+Feature:json write
+
+
+Scenario Outline:
```
Comment (SubithaMS) [inline json_write.feature]: description missing

File: json_write.feature [hunk 4]
```diff
@@ -0,0 +1,21 @@
+Feature:json write
+
+
```
Comment (SubithaMS) [inline json_write.feature:3]: one new line is expected for good formating

File: json_write.feature [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+Scenario Outline:
+    * def jsonObject = <data>
+    * string filename = <filename>
+    # * def new = function(){ return karate.write(data,"main.json")}
+    # * def FileUtils = Java.type('com.intuit.karate.FileUtils')
+    # * def result = FileUtils.writeToFile('temp.txt','hello world')
```
Comment (SubithaMS) [inline json_write.feature]: remove unwanted codes

**Review by SubithaMS — APPROVED** [review]

File: json_write.feature (hunk 5 above)
Comment (santhosh-as-rsk) [reply json_write.feature]: cleared

**Pull request merged**